src/lib.rs

```diff
@@ -26,7 +26,7 @@ use core::fmt;
 pub use bounds::Bounded;
 #[cfg(feature = "std")]
 pub use float::Float;
-pub use float::FloatConst;
+pub use float::{CoreFloat, FloatConst};
 // pub use real::Real; // NOTE: Don't do this, it breaks `use num_traits::*;`.
```

Comment:
I think we may have the same concern as `Real` did here.  If someone does `use num_traits::*`, then the common methods between `Float` and `CoreFloat` may be ambiguous.  Would you mind if we didn't re-export this in the root?

Reply:
I'm not sure supporting wildcard imports is worthwhile. If you want to make sure not to break them, you can't add any new functions or types. However, in this case I guess it is easy enough to avoid that problem.

Reply:
I'm not sure either.  I guess a wildcard making us self-conflicting is a worse than having a conflict with some unrelated crate though.